### PR TITLE
[RU] Update the `Contribution guide`, `Best practices` of contribution, and add the `github.dev` guide

### PR DESCRIPTION
**Pull request by TicClick**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/osu!_wiki/Contribution_guide/ru.md [hunk 1]
```diff
@@ -1,113 +1,66 @@
----
-outdated: true
-outdated_since: 1c921bb312848bb2dce76217542281d0db5a6825
----
-
 # Руководство по работе с osu! wiki
 
 Спасибо за интерес, проявленный к osu! wiki.
```
[the remark this answers is not in the document]
Comment (TicClick) [reply wiki/osu!_wiki/Contribution_guide/ru.md:3]: that wasn't my intention, need to take a second look at what I've done

File: wiki/osu!_wiki/Contribution_guide/ru.md [hunk 2]
```diff
@@ -1,113 +1,66 @@
...
+Эта статья адресована тем, кто прежде не работал с GitHub и не участвовал в редактировании osu! wiki. Если такой опыт имеется, пользуйтесь схемой «одна статья (или группа статей на одну тематику) — одна ветка», а потом сразу переходите к разделу «[Самопроверка](#самопроверка)».
 
-*Примечание: на самом деле, инструментов для работы с git и GitHub намного больше, но здесь они не упомянуты ради упрощения статьи.*
+Если вам понадобится помощь, обращайтесь в [дискорд-сервер osu!dev](https://discord.gg/ppy), канал `#osu-wiki`.
 
-**После знакомства с одним из способов редактирования переходите к следующему разделу.**
+## Чем можно помочь
 
-## Публикация изменений
+Если у вас есть желание помочь, но нет понимания, с чем именно, загляните в раздел [Работа над osu! wiki § Рутинные задачи](/wiki/osu!_wiki/Maintenance#рутинные-задачи). В ней описаны вещи, которые нуждаются в регулярном контроле, а также прочие способы помощи. Если вы пытаетесь понять, как тут работает форматирование, посмотрите [шпаргалку по Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
```
Comment (UristMcKobold) [inline wiki/osu!_wiki/Contribution_guide/ru.md:11]:
```suggestion
Если у вас есть желание помочь, но нет понимания, с чем именно, загляните в раздел [Работа над osu! wiki § Рутинные задачи](/wiki/osu!_wiki/Maintenance#рутинные-задачи). В ней описаны задачи, которые нуждаются в регулярном контроле, и прочие способы помочь. Если вы пытаетесь понять, как тут работает форматирование, посмотрите [шпаргалку по Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
```

File: wiki/osu!_wiki/Contribution_guide/ru.md [hunk 3]
```diff
@@ -1,113 +1,66 @@
...
 
-Сразу после создания пулл-реквеста его смогут увидеть и прокомментировать другие люди, указав на возможные недочёты. Пулл-реквест, где долгое время ничего не происходит, могут закрыть, поэтому ваша задача — **следить за комментариями, отвечать на них и исправлять найденные ошибки.** Если вы хотите, чтобы вашу работу проверили, напишите в [дискорд osu!dev](https://discord.gg/ppy) на канал `#osu-wiki` или в комментарии на GitHub.
+## Публикация
 
-### Слияние изменений
+Рано или поздно ваш пулл-реквест проверит один из [администраторов вики](/wiki/osu!_wiki/Maintenance/List_of_maintainers) – обычно это происходит после ревью от других людей. Если ничего не происходит в течение нескольких дней, призовите администраторов в комментариях к пулл-реквесту, либо на канале `#osu-wiki` в [дискорд-сервере osu!dev](/wiki/Community/osu!dev_Discord_server). После того, как вашу ветку вольют в `master`, соответствующие статьи обновятся (чаще всего это происходит за пару секунд, в худшем случае – в течение пяти часов).
```
Comment (UristMcKobold) [inline wiki/osu!_wiki/Contribution_guide/ru.md:64]:
```suggestion
Рано или поздно ваш пулл-реквест проверит один из [администраторов вики](/wiki/osu!_wiki/Maintenance/List_of_maintainers) – обычно это происходит после ревью от других людей. Если ничего не происходит в течение нескольких дней, позовите администраторов в комментариях к пулл-реквесту, либо на канале `#osu-wiki` в [дискорд-сервере osu!dev](/wiki/Community/osu!dev_Discord_server). После того, как вашу ветку зальют в `master`, соответствующие статьи обновятся (чаще всего это происходит за пару секунд, в худшем случае – в течение пяти часов).
```

Comment (TicClick) [reply wiki/osu!_wiki/Contribution_guide/ru.md:64]: «вольют» — это терминология (хоть и кривая, потому что на самом деле это **сли**яние), с первым согласен

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**